app/docs/0.13.x/auth.md

```diff
@@ -36,49 +36,56 @@ It is an opaque concept to Kong and hence they are called "consumers" and not "u
 
 ## Anonymous Access
 
-Prior to Kong 0.10.x, a given API could be configured to allow **only** authenticated access (by applying an
-[auth plugin][plugins]) or **only** anonymous access - it was not possible to have a given API allow some 
-users to be authenticated and others to access anonymously.
+Kong has the ability to configure a given Service to allow **both** authenticated **and** anonymous access.
+You might use this configuration to grant access to anonymous users with a low rate-limit, and grant access
+to authenticated users with a higher rate limit.
 
-Kong 0.10.x adds the ability to configure a given API to allow **both** authenticated **and** anonymous access. 
-You might use this configuration to grant access to anonymous users with a low rate-limit, and grant access 
-to authenticated users with a higher rate limit. 
+To configure a Service like this, you first apply your selected authentication plugin, then create a new
+consumer to represent annonymous users, then configure your authentication plugin to allow anonymous
+access. Here is an example, which assumes you have already configured a Service named `example-service` and
+the corresponding route:
 
-To configure an API like this, you first apply your selected authentication plugin, then create a new 
-consumer to represent annonymous users, then configure your authentication plugin to allow anonymous 
-access. Here is an example, which assumes you have already configured an API named `example-api`:
+1. ### Create an example Service and a Route
 
-1. ### Create an example API
-
-    Issue the following cURL request to create `example-api` which will use the
+    Issue the following cURL request to create `example-service` which will use the
```

Comment:
This becomes really confusing now that we refer to both the Kong Service and "the mockbin service" in the same sentence...